app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/OnlinePaymentServiceTemplates.scala

```diff
@@ -51,22 +51,6 @@ object OnlinePaymentServiceTemplates {
 
     // Direct Debit templates
 
-    MessageTemplate.create(
-      templateId = "ddi_cancelled",
-      fromAddress = FromAddress.noReply("VAT Direct Debit"),
-      service = ServiceIdentifier.OnlinePaymentService,
-      subject = "VAT Return Direct Debit: Cancelled Direct Debit",
-      plainTemplate = txt.ddi_cancelled.f,
-      htmlTemplate = html.ddi_cancelled.f,
-      priority = Some(MessagePriority.Urgent)),
-    MessageTemplate.create(
-      templateId = "ddi_advance_notice",
```

Comment:
Again, may need this still for the Advance Notice For Hybrid Users that's sent directly as an email rather than via DCS